gridstatus/ercot_api/ercot_api.py

```diff
@@ -798,6 +803,7 @@ def get_hourly_resource_outage_capacity(self, date, end=None, verbose=False):
             end_date=end,
             verbose=verbose,
             add_post_datetime=True,
+            bulk_download=True,
```

Comment:
This is now the default (added [last week](https://github.com/gridstatus/gridstatus/pull/508)). I'm fine with being explicit too.